packages/core/src/amazonqGumby/chat/controller/messenger/stringConstants.ts

```diff
@@ -7,7 +7,7 @@
 export const enterJavaHomeMessage = 'Enter the path to JDK '
 
 export const windowsJavaHomeHelpMessage =
-    'To find the JAVA_HOME path, run the following command in a new IDE terminal: `cd C:ProgramFilesJava` and then `dir`. If you see your JDK version, run `cd JDK<version>` and then `cd` to show the path.'
+    'To find the JAVA_HOME path, run the following command in a new IDE terminal: `cd "C:\\Program Files\\Java" && dir`. If you see your JDK version, run `cd <version>` and then `cd` to show the path.'
```

Comment:
This is never a reason to use `\` path separators on Windows. Just use "/" everywhere.
```suggestion
    'To find the JAVA_HOME path, run the following command in a new IDE terminal: `cd "C:/Program Files/Java" && dir`. If you see your JDK version, run `cd <version>` and then `cd` to show the path.'
```